global.json

```
@@ -1,4 +1,8 @@
 {
+  "sdk": {
+    "version": "7.0.116",
+    "rollForward": "disable"
```

nit, to be more consistent w/ at least release/6.0 (where we've avoided this issue):
```suggestion
    "rollForward": "latestFeature"
```

---

I also suggest porting this update to release/8.0 and main. we'll be scratching our heads again too soon otherwise